nextflow_schema.json

```diff
@@ -99,6 +99,78 @@
+                    "fa_icon": "fas fa-border-none"
+                },
+                "annotation_prokka_mincontiglen": {
+                    "type": "integer",
+                    "default": 1,
+                    "description": "Limit the contigs lengths.",
+                    "help_text": "Limit the contig lengths to only carry out annotations on contigs >= the assigned length. This should not be >200 as NCBI can only process contigs >= 200.\n\n> Modifies tool parameter(s):\n> - PROKKA: `--mincontiglen`",
+                    "fa_icon": "fas fa-ruler-horizontal"
+                },
+                "annotation_prokka_evalue": {
+                    "type": "number",
+                    "default": 1e-6,
+                    "description": "Minimum e-value cut-off.",
+                    "help_text": "Assigns the minimum e-value used for filtering the alignment hits.\n\n> Modifies tool parameter(s):\n> - PROKKA: `--evalue`",
+                    "fa_icon": "fas fa-sort-amount-down"
+                },
+                "annotation_prokka_coverage": {
+                    "type": "integer",
+                    "default": 80,
+                    "description": "Set the assigned minimum coverage.",
+                    "help_text": "Specifies the minimum coverage of the annotated genome. This must be set between 0-100. \n\n> Modifies tool parameter(s):\n> - PROKKA: `--coverage`",
+                    "fa_icon": "fas fa-align-right"
+                },
+                "annotation_prokka_cdsrnaolap": {
+                    "type": "boolean",
+                    "description": "Allow transfer RNA (trRNA) to overlap coding sequences (CDS). Transfer RNAs are short streatches of nucleotide sequences that link mRNA and teh amino acid sequence of proteins. Their presence helps in the annotation of the sequences, because each trRNA can only be attached to one type of amino acid.",
+                    "help_text": "Allow transfer RNA (trRNA) to overlap coding sequences (CDS).\n\n> Modifies tool parameter(s):\n> - PROKKA: `--cdsrnaolap`",
+                    "fa_icon": "fas fa-align-justify"
+                },
+                "annotation_prokka_rnammer": {
+                    "type": "boolean",
+                    "description": "Use RNAmmer for rRNA prediction.",
+                    "help_text": "Activates RNAmmer instead of the PROKKA default Barnap for rRNA prediction during teh annotation process. RNAmmer classifies ribosomal RNA genes in genome sequences by using two levels of Hidden Markov Models. Barnap uses nhmmer tool that includes HMMER 3.1 for HMM searching in RNA:DNA style. More info about 'Barnap' can be found [here](https://github.com/tseemann/barrnap).\n\n> Modifies tool parameter(s):\n> - PROKKA: `--rnammer`",
```

Comment:
```suggestion
                    "help_text": "Activates RNAmmer instead of the PROKKA default Barnap for rRNA prediction during the annotation process. RNAmmer classifies ribosomal RNA genes in genome sequences by using two levels of Hidden Markov Models. Barnap uses the nhmmer tool that includes HMMER 3.1 for HMM searching in RNA:DNA style. More info about 'Barnap' can be found [here](https://github.com/tseemann/barrnap).\n\n> Modifies tool parameter(s):\n> - PROKKA: `--rnammer`",
```